trinity/protocol/common/managers.py

```diff
@@ -33,6 +34,27 @@
 TReturn = TypeVar('TReturn')
 
 
+class ResponseTimeTracker:
+
+    def __init__(self) -> None:
+        self.total_msgs = 0
+        self.total_items = 0
+        self.total_response_time = 0.0
+
+    def get_stats(self) -> str:
+        if not self.total_msgs:
+            return 'None'
+        avg_rtt = self.total_response_time / self.total_msgs
+        per_item_rtt = self.total_response_time / self.total_items
```

Comment:
This is succeptable to division by zero if the peer is returning empty responses.

Reply:
I've changed the stats to include the total number of items as well, and use 0 for per_item_rtt when self.total_items is 0